lib/datadog/tracing/contrib/ethon/easy_patch.rb

```diff
@@ -168,6 +183,26 @@ def analytics_enabled?
             def analytics_sample_rate
               datadog_configuration[:analytics_sample_rate]
             end
+
+            # `#response_headers` returns a "\n" concatenated String containing:
+            # * The HTTP Status-Line.
+            # * The response headers.
+            # * A trailing "\n".
+            #
+            # This method extracts only the headers from it.
+            def parse_response_headers
```

Comment:
Is this method tested somewhere? 

Our ethon integration test seems to provide response_headers as a hash, which probably is not covering the implementation of this method.

Reply:
This method is tested in `spec/datadog/tracing/contrib/ethon/integration_test_spec.rb`.

There was actually no test for `response_headers` itself, this is the first time we are testing real header usage for Ethon.